dotnet/src/webdriver/IFileDetector.cs

```diff
@@ -16,6 +16,10 @@
 // limitations under the License.
 // </copyright>
 
+#nullable enable
```

Comment:
Let's put nullable directive after using statements.

Reply:
Fixed in all files